src/actions/users.js

```diff
@@ -79,15 +79,15 @@ export function createUser(user, data, hash, inviteId) {
     };
 }
 
-export function login(loginId, password, mfaToken = '') {
+export function login(loginId, password, mfaToken = '', ldapOnly = false) {
     return async (dispatch, getState) => {
         dispatch({type: UserTypes.LOGIN_REQUEST}, getState);
 
         const deviceId = getState().entities.general.deviceToken;
 
         let data;
         try {
-            data = await Client4.login(loginId, password, mfaToken, deviceId);
+            data = await Client4.login(loginId, password, mfaToken, deviceId, ldapOnly);
```

Comment:
It's something the webapp has an action for, not exactly sure where it's used. I don't think you need to worry about it on RN. I just wanted to add the support here since the server allows it